examples/crypto-trading-basic.ipynb

```diff
@@ -4,7 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Alpaca-py crypto trading basic"
+    "# Alpaca-py crypto trading basic\n",
+    "\n",
+    "Notice: This notebook is moved [to crypto-trading-basic.ipynb under crypto directory](https://github.com/alpacahq/alpaca-py/blob/master/examples/crypto/crypto-trading-basic.ipynb)\n",
+    "\n",
+    "Please update your bookmark/link to the new location. This redirect page will be removed after 2026-03-01.\n"
```

Comment:
Could you please remove codes below and keep this message in the existing noteboks?